subworkflows/local/bam_variant_calling_somatic_muse/main.nf

```diff
@@ -0,0 +1,76 @@
+//
+// MuSE tumor-normal variant calling
+//
+// For all modules here:
+// A when clause condition is defined in the conf/modules.config to determine if the module should be run
+
+include { MUSE_CALL } from '../../../modules/nf-core/muse/call/main'
+include { MUSE_SUMP } from '../../../modules/nf-core/muse/sump/main'
+include { SAMTOOLS_CONVERT as CRAM_TO_BAM_TUMOR } from '../../../modules/nf-core/samtools/convert/main'
+include { SAMTOOLS_CONVERT as CRAM_TO_BAM_NORMAL } from '../../../modules/nf-core/samtools/convert/main'
+
+workflow BAM_VARIANT_CALLING_SOMATIC_MUSE {
+    take:
+    cram          // channel: [mandatory] [ meta, normal_cram, normal_crai, tumor_cram, tumor_crai ]
+    fasta         // channel: [mandatory] [ meta, fasta ]
+    fai           // channel: [mandatory] [ meta, fai ]
+    dbsnp         // channel: [optional] [ dbsnp ]
+    dbsnp_tbi     // channel: [optional] [ dbsnp_tbi ]
+
+    main:
+    versions = Channel.empty()
+
+    CRAM_TO_BAM_TUMOR(
+        cram.map{ meta, normal_cram, normal_crai,tumor_cram, tumor_crai -> [ meta + [ tobam: 'tumor' ], tumor_cram, tumor_crai ] },
+        fasta,
+        fai
+    )
+
+    CRAM_TO_BAM_NORMAL(
+        cram.map{ meta, normal_cram, normal_crai,tumor_cram, tumor_crai -> [ meta + [ tobam: 'normal' ], normal_cram, normal_crai ] },
+        fasta,
+        fai
+    )
```

Comment:
we should think of a more general strategy, we now have a couple of tools that require BAM files instead. if someone would run cnvkit and muse right now they would do the conversion twice which is wasteful

Reply:
I suppose there is no way the muse dev would add cram support?

Reply:
I can ask the team for sure but I would not wait for them with this PR. I can try and do the conversion for both cnvkit and muse -> are there any other tools that also need this?